src/createServer.js

```diff
@@ -1,3 +1,36 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { getErrors } = require('./getErrors');
+
+const createServer = () => {
+  return http.createServer((request, response) => {
+    const [text, searchParams] = request.url.slice(1).split('?');
+    const params = new URLSearchParams(searchParams);
+    const toCase = params.get('toCase');
+    const errors = getErrors(text, toCase);
```

Comment:
I would use validate in this context. get it's when you just retrieve information, but here you are checking request params, so name can be `validateRequest` or `validateRequestParams`